src/test/java/seedu/storemando/logic/commands/AddCommandTest.java

```diff
@@ -42,6 +42,19 @@ public void execute_itemAcceptedByModel_addSuccessful() throws Exception {
         assertEquals(Arrays.asList(validItem), modelStub.itemsAdded);
     }
 
+    @Test
+    public void execute_itemAcceptedByModel_addExpiredSuccessful() throws Exception {
```

Comment:
U could try using assert throw like in the test right below this. U could also specify what the Exception you are throwing actually is.

Reply:
Fixed.